src/Tizen.NUI/src/public/Common/Color.cs

```diff
@@ -1232,7 +1233,12 @@ public float A
         /// <since_tizen> 3 </since_tizen>
         public static implicit operator Vector4(Color color)
         {
-            return new Vector4((float)color?.R, (float)color.G, (float)color.B, (float)color.A);
+            if (color == null)
+            {
+                Tizen.Log.Error("NUI", "color is null");
```

Comment:
I don't think this is an error case to show an error message.
```C#
Color color = null;
Vector4 vector4 = color;
```
This can happen sometimes.

Reply:
delete all logs